_data/simple-icons.json

```
@@ -5155,6 +5155,14 @@
             "source": "https://www.firstinspires.org/brand",
             "guidelines": "https://www.firstinspires.org/brand"
         },
+        {
+            "title": "fish shell",
+            "hex": "4ABC39",
```

In the EPS file, the color is `#34C534`. Where did you got this one?
```suggestion
            "hex": "34C534",
```

---

Huh, I thought I pulled it from the SVG! Sorry about that.